.changelog/261.internal.md

```diff
@@ -1 +1 @@
-ci: update gitlint config
```

Comment:
Please, create a new Change Log fragment file for this, e.g. `261.internal.2.md`.